changelog/0.33.md

```diff
@@ -0,0 +1,77 @@
+- [#1707](https://github.com/elastisys/compliantkubernetes-apps/pull/1707) - apps sc: move rclone from kube-system @lunkan93
+- [#1708](https://github.com/elastisys/compliantkubernetes-apps/pull/1708) - apps sc: rclone-sync migration script @linus-astrom
+- [#1709](https://github.com/elastisys/compliantkubernetes-apps/pull/1709) - apps: falco-exporter chart v0.9.6 @OlleLarsson
+- [#1710](https://github.com/elastisys/compliantkubernetes-apps/pull/1710) - template: add new checks for pull request @crssnd
+- [#1711](https://github.com/elastisys/compliantkubernetes-apps/pull/1711) - scripts: some fixes for the clean-up scripts @crssnd
+- [#1713](https://github.com/elastisys/compliantkubernetes-apps/pull/1713) - apps sc: grafana user values file @crssnd
+- [#1717](https://github.com/elastisys/compliantkubernetes-apps/pull/1717) - Patches from release 0.32 @lunkan93
+- [#1719](https://github.com/elastisys/compliantkubernetes-apps/pull/1719) - tests: test script all at once option @linus-astrom
+- [#1722](https://github.com/elastisys/compliantkubernetes-apps/pull/1722) - apps sc: Refer to Grafana, OpenSearch and Harbor as Web Portals @jakubkrzywda
+- [#1725](https://github.com/elastisys/compliantkubernetes-apps/pull/1725) - apps sc & wc: Increased the default proxy-buffer-size to 8k @raviranjanelastisys
+- [#1726](https://github.com/elastisys/compliantkubernetes-apps/pull/1726) - fix typo in rclone migration @lunkan93
+- [#1727](https://github.com/elastisys/compliantkubernetes-apps/pull/1727) - apps: separate capacity alert limits for pv @anders-elastisys
+- [#1728](https://github.com/elastisys/compliantkubernetes-apps/pull/1728) - apps sc: rclone sync changes @crssnd
+- [#1731](https://github.com/elastisys/compliantkubernetes-apps/pull/1731) - docs: fix markdown syntax for notes @Elias-elastisys
+- [#1732](https://github.com/elastisys/compliantkubernetes-apps/pull/1732) - Crssnd/grafana expose additional configs @crssnd
+- [#1735](https://github.com/elastisys/compliantkubernetes-apps/pull/1735) - apps sc & wc: Fixed the csi-upcloud networkpolicy template @raviranjanelastisys
+- [#1742](https://github.com/elastisys/compliantkubernetes-apps/pull/1742) - apps:Replace elastisys/curl-jq:latest @lucianvlad
+- [#1744](https://github.com/elastisys/compliantkubernetes-apps/pull/1744) - apps: Only mutate pods on create @aarnq
+- [#1747](https://github.com/elastisys/compliantkubernetes-apps/pull/1747) - apps sc: add security context for the curl-jq images @crssnd
+
+## v0.33.1
+
+Released 2023-10-20
+
+> [!WARNING]
+> **Security Notice(s)**
+> - New curl release (CVE-2023-38545 and CVE-2023-38546)
+>   New Go release (https://github.com/advisories/GHSA-qppj-fm5r-hxr3 and https://github.com/advisories/GHSA-4374-p667-p6c8)
+>   Fix for HTTP/2 Rapid Reset Attack [CVE-2023-44487](https://nvd.nist.gov/vuln/detail/CVE-2023-44487)
```

Comment:
We need to better look at the PRs to see that it is formatted for regular markdown files :sweat_smile:

Reply:
Yea that's probably going to be difficult to keep track of. Maybe if we automatically generated the changelog and put up a preview in the PR comments for each push but other than that people won't consider it I think.

Reply:
At least it's not a big deal to fix manually: https://github.com/elastisys/compliantkubernetes-apps/pull/1867/commits/5df37de5bac035ec858bc31e7241305588f038f2